core/modules/views/includes/view.inc

```diff
@@ -2003,6 +2009,11 @@ class view {
       // property set to FALSE, signifying localization should not occur.
       if (empty($this->localization_plugin)) {
         $this->localization_plugin = views_get_plugin('localization', 'none');
+        // If it is still NULL not even plugin "none" is available. Only needed
+        // for edge cases like core update.
```

Comment:
Some punctuation is needed here:

```suggestion
        // If it is still NULL, not even plugin "none" is available. Only needed
        // for edge cases, like core update.
```

See:
- https://www.grammarly.com/blog/conditional-sentences "Use a comma after the if-clause when the if-clause precedes the main clause."
- https://medium.com/@Ediket/when-do-you-use-a-comma-before-including-or-such-as-9b3e1b4f7af3 for adding a comma before things like "including", or "such as", or "like".

Reply:
I'll just add your commas, I don't want to actually learn punctuation rules in a foreign language. No need for a English course here. :rofl: